### ISSUE TITLE
[BUG] f5_journeys not compatible with Apple Silicon (M2) (linux/arm64/v8)

### ISSUE THREAD

**Issue by jewettg** [f5devcentral/f5-journeys#117]
**Describe the bug**
This bug roles together the following issues: #115 and #99.

**To Reproduce**
1. Perform the steps outlined within [Journey's Installation](https://github.com/f5devcentral/f5-journeys#journeys-setup-requirements).
2. After step 6, `docker-compose up -d` you receive the following warnings.
```
[+] Running 9/9
 ✔ Network f5-journeys_default                                                                                                                                  Created                         0.0s
 ✔ Container f5-journeys-acc-api-1                                                                                                                              Started                         0.3s
 ✔ Container f5-journeys-postgres-1                                                                                                                             Started                         0.3s
 ✔ Container f5-journeys-redis-1                                                                                                                                Started                         0.3s
 ! acc-api The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested                                       0.0s
 ✔ Container f5-journeys-journeys-1                                                                                                                             Started                         0.4s
 ! journeys The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested                                      0.0s
 ✔ Container f5-journeys-celery-worker-1                                                                                                                        Started                         0.6s
 ! celery-worker The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested                                 0.0s
```
**This references issues: #99** 
3. Access the web interface and attempt to upload a recent copy of the UCS file using either option, and you get the following error:
![image009](https://github.com/f5devcentral/f5-journeys/assets/17208818/f3083233-ad5c-48ee-bd3a-037e82949bae)
**This references #115.**

**Expected behavior**
It would be expected that no warnings would appear and a UCS file would upload.  However, based on previous reports, it appears that the code is hard-locked to a specific platform and using the linux/arm64/v8m  (Apple Silicon M2) will not work.  There are numerous other issues with vulnerabilities (CVEs) in many of the included packages, so those should also be addresses as well.

**Screenshots**
![image010](https://github.com/f5devcentral/f5-journeys/assets/17208818/38b2b20d-0c70-45f2-850c-6177380d4c80)

**Context**
- Apple MacBook Pro M2 Laptop
- Mac OS Ventura 13.5.2
- Docker Desktop 4.23.0 (120376)
  - Engine: 24.0.6
  - Compose: v2.21.0-desktop.1
  - Kubernetes: v1.27.2
- f5-bigip-journeys-app v3.3.5
  - f5devcentral/f5-automation-config-converter:1.23.0

**Comment by jewettg** [comment]
From #99, it was suggested to add the line `platform: "linux/am64/v8"` into three different places within `docker-compose.yml` file:  services/acc-api; services/journeys; and services/celery-worker.

If this is done, the `docker-compose pull` command will fail with the following line:
`image with reference docker.io/f5devcentral/f5-automation-config-converter:1.23.0 was found but does not match the specified platform: wanted linux/am64/v8, actual: linux/amd64`

**Comment by marlonfrank30** [comment]
hi Greg,
speaking with other coworkers here at F5 and digging deeper into it, they believe it won't be supported anytime soon I am afraid.. some of the components used on Journeys already support the new architecture however Journeys itself doesn't as  you can see the supported architectures below:
![image](https://github.com/f5devcentral/f5-journeys/assets/55821050/0c4ad478-934a-45f7-b11e-264c45ffda09)

**Comment by f5-purc** [comment]
Thanks for your comments, we're working on an update for 17.1 support that should be out soon and later we'll add other required fixes and improvements.

**Comment by rmaliszewskif5** [comment]
Duplicate of #99
